src/pages/PatientIntakeForm.js

```diff
@@ -24,16 +24,18 @@ const PatientIntakeForm = () => {
   const getPatientDetail = async (hashKey) => {
     try {
       const response = await patientService.getPatientDetails(hashKey);
-      getPatientIntake(response.patientInfo.patientId);
+      getPatientIntake(response.patientInfo.patientId, response.patientInfo.messageType);
       setPatientDatails(response.patientInfo);
       setIsAgreed(response.patientInfo.isAgreed);
     } catch (err) {}
   };
 
-  const getPatientIntake = async (patientId) => {
+  const getPatientIntake = async (patientId, messageType) => {
     try {
+      if(messageType !== MESSAGE_TYPES.webMeeting){
       const response = await patientService.getPatientIntake(patientId);
       setIntakeForm(JSON.parse(response));
+      }
```

Comment:
NIT: since this function has a `if` condition and if the condition is false, it does nothing, it is better to put the `if` condition where the function is being called instead of inside here

Reply:
this will be handled in the other PR with same comment